File: PetHolidayBackend/Domain/Models/PetSitterProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class PetSitterProfile
    {
        public int ID { get; set; }
        public string? Description { get; set; }
        public int AcquiredExperience { get; set; }
        
        public int UserID { get; set; }
        public virtual User User { get; set; } = null!;
    }
}

File: onlabFrontend/src/models/PetSitterProfile.ts
export interface PetSitterProfile {
  id: number;
  description?: string;
  acquiredExperience?: number;
}

export interface UpdatePetSitterProfileModel {
  description?: string;
  acquiredExperience?: number;
}

export const defaultUpdatePetSitterProfileModel: UpdatePetSitterProfileModel = {
  description: "",
  acquiredExperience: 0,
};
